source/installation/config.html.md

```diff
@@ -43,18 +44,41 @@ MAIL_USER=example
 MAIL_PASSWORD=123456
 MAIL_HELO_NAME=example.org
 MAIL_ADDRESS=from@example.org
+
+# Custom icons
+#PLUME_LOGO=icons/custom/plume.png
+#PLUME_LOGO_FAVICON=icons/custom/plume32.png
+#PLUME_LOGO_48=icons/custom/plume48.png
+#PLUME_LOGO_72=icons/custom/plume72.png
+#PLUME_LOGO_96=icons/custom/plume96.png
+#PLUME_LOGO_144=icons/custom/plume144.png
+#PLUME_LOGO_160=icons/custom/plume160.png
+#PLUME_LOGO_192=icons/custom/plume192.png
+#PLUME_LOGO_256=icons/custom/plume256.png
+#PLUME_LOGO_512=icons/custom/plume512.png
 ```
 
 For more information about what you can put in your `.env`,
 see [the documentation about environment variables](/environment).
 
 Now we need to run migrations. Migrations are scripts used to update
-the database. To run the migrations, you can do:
+the database. To run the migrations, you can do for **GNU/Linux** and **Mac OS X**:
 
 ```bash
 plm migration run
 ```
 
+If you are using **Windows** and DATABASE of sqlite, you will need to copy the sqlite3.dll from 
+"C:\ProgramData\chocolatey\lib\SQLite\tools" to where plm.exe and plume.exe were compiled:
+```
+copy "C:\ProgramData\chocolatey\lib\SQLite\tools\sqlite3.dll" "C:\Users\%USERNAME%\.cargo\bin\sqlite3.dll"
+```
+
+Now you may run the migrations:
+```
+diesel migration run
```

Comment:
Isn't it with `plm` that the migrations are run? (`diesel` should work too, but it means one more command to install)

Reply:
In 0.3, I had lots of problems doing `plm` verses `diesel`. I can try building again but using `plm` this time.

Reply:
Yes, diesel VS plm was a mess in 0.3, but it should be fine with 0.4